packages/react-components/react-components/src/index.ts

```diff
@@ -1005,6 +1005,7 @@ export {
   useTableSelection,
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29780&commitId=c1d93eb74574a8c9e50ef90868ea73c63e27989d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.visibility+focus.focus closed.chromium.png|982| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|824| Changed|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|982| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|673| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>